end_to_end_testing/basic.yml

```diff
@@ -0,0 +1,26 @@
+namespace: test #namespace to install the chart in, defaults to 'kube-system'
+repositories:
+  test_repo:
+    url: https://kubernetes-charts.storage.googleapis.com
+  incubator:
+    url: https://kubernetes-charts-incubator.storage.googleapis.com
+  reactiveops-stable:
+    url: https://charts.reactiveops.com/stable
+  reactiveops-incubator:
+    url: https://charts.reactiveops.com/incubator
+minimum_versions: #set minimum version requirements here
+  helm: 0.0.0
+  reckoner: 0.0.0
+charts:
```

Comment:
It would be super simple to add a couple hooks here that will add testing for that path. 

I know there's further expansion planned, but this would be a nice quick win.

Reply:
:+1:

Reply:
So I don't think reckoner returns a bad status code when hooks fail - it just skips the chart. I'll have to test to make sure the test would be valid - at least it would break even if it would cost some debugging :D

Reply:
I think that relates a bit to #86.  Definitely not a blocker for this PR.

Reply:
I did get an example running but it definitely isn't testing in enough isolation - MVP :D at least we've got something for a start. Still could have false positives and false negatives. We'll keep improving as we find bugs.